frontend/website/docs/coda-testnet.md

```diff
@@ -16,99 +16,99 @@ You can track the Testnet Points[\*](#disclaimer) and current leaders in [the pu
+Bring out your most creative self to create Coda-related GIFs and emoji's! Post your GIF or emoji on the [forum](https://forums.codaprotocol.com/t/community-art-contest-gifs/109). You can have unlimited number of entries so cut yourself loose! The community can vote on the best entries by “hearting” your post, so do not forget to "heart" your favorite entries! Top 3 entries will receive bonus points: 300 pts[\*](#disclaimer) for the best GIF and emoji, 200 pts[\*](#disclaimer) for the second place and 100 pts[\*](#disclaimer) for the third place.
 
 **Challenge #15 (expired): "Something Snarky"**
 
-This week's challenge is to earn testnet tokens by performing SNARK work. Node operators that produce at least one SNARK will get 1000 pts\*.
+This week's challenge is to earn testnet tokens by performing SNARK work. Node operators that produce at least one SNARK will get 1000 pts[\*](#disclaimer).
 
-BONUS : Top 3 node operators who are able to earn the most tokens with their SNARKs on the snarketplace will win an additional +3000 pts\* , +2000 pts\* , and +1000 pts\* respectively. Hint — your SNARKs are more likely to be bought if the fees are lower, so feel free to experiment with the economics!
+BONUS : Top 3 node operators who are able to earn the most tokens with their SNARKs on the snarketplace will win an additional +3000 pts[\*](#disclaimer) , +2000 pts[\*](#disclaimer) , and +1000 pts[\*](#disclaimer) respectively. Hint — your SNARKs are more likely to be bought if the fees are lower, so feel free to experiment with the economics!
 
-**Challenge #16: "New Member Bonus"**
+**Challenge #16 (on-going): "New Member Bonus"**
 
-This week, we have a package deal with a big bonus for new members. This would be a great chance to invite your friends to join! New members who complete *all* of the following three challenges this week will receive **two times the total points* value as a bonus**: #1 'Connect to Testnet', #3 'Join Discord', and #6 'Nice to Meet You' (check out all challenge descriptions [here](/docs/coda-testnet/)). So instead of 700 pts\* (respectively 500 + 100 + 100), new users will receive **1400 pts**\* ! - [Get started!](/docs/getting-started/)
+This week, we have a package deal with a big bonus for new members. This would be a great chance to invite your friends to join! New members who complete all of the following three challenges this week will receive two times the total points[\*](#disclaimer) value as a bonus: #1 'Connect to Testnet', #3 'Join Discord', and #6 'Nice to Meet You' (check out the individual challenge descriptions above). So instead of 700 pts[\*](#disclaimer) (respectively 500 + 100 + 100), new users will receive 1400 pts[\*](#disclaimer) ! 
 
-**Challenge #17: "Hello Memo"**
+**Challenge #17 (on-going): "Hello Coda"**
```

Comment:
I think Hello Memo was right?

Reply:
I did not change the challenge name. But I also agree that it should be Hello Memo. I have Hello Memo all over the place in the testnet communications.